src/xgboost_lib.jl

```diff
@@ -294,33 +300,78 @@ function aggcv(rlist; show_stdv = true)
     end
     items = [item for item in cvmap]
     sort!(items, by = x -> x[1])
+
+    res = [ret]
+
     for item in items
         k = item[1]
         v = item[2]
+
         if show_stdv == true
-            ret *= @sprintf("\tcv-%s:%f+%f", k, mean(v), std(v))
+            res = [res k mean(v)]
         else
-            ret *= @sprintf("\tcv-%s:%f", k, mean(v))
+            res = [res k mean(v) std(v)]
         end
     end
-    return ret
+
+    res
 end
 
 function nfold_cv(data, num_boost_round::Integer = 10, nfold::Integer = 3; label = Union{},
                   param=[], metrics=[], obj = Union{}, feval = Union{}, fpreproc = Union{},
                   show_stdv = true, seed::Integer = 0, kwargs...)
     dtrain = makeDMatrix(data, label)
-    results = String[]
+    results = []
```

Comment:
It might be not the most efficient way, but the reason I am having a vector of any is to actually split the results from a string to multiple columns. These way we will be be able to work with the results and stop when test error is at its minimum.